src/modules/feedback/components/__tests__/FeedbackBoxContainer.spec.js

```diff
@@ -356,6 +364,7 @@ describe('FeedbackBoxContainer', () => {
         extras={null}
         postFeedbackDataOverride={mockPostFeedbackData}
         theme={theme}
+        feedbackSent='SUCCESS'
```

Comment:
could you add one test where submitting feedback fails?
And when looking at the tests it feels stange that feedback sent is a property of the FeedbackBoxContainer, as it is represents more the State of the container I think. What was the reasonin behind modelliin it as a property?

Reply:
Regarding the test I'd like to talk to you.
Regarding the sending status as a prop, I thought to place the error feedback text at the box component between the button and the text field. Therefor the prop seemes to be useful.

Reply:
@Taggotty The specs has been completed ;)

Reply:
@klinzo The Flow type monitor `src/modules/feedback/components/__tests__/FeedbackBoxContainer.spec.js:391:5` cannot assign function to `console.error` because property `error` is not witeable. Any suggstions?